### PR TITLE
Enable login while ldap is offline

### PR DESCRIPTION
**Pull request by forevermatt**

### CODE REVIEW

File: dictionaries/error.definition.json [hunk 1]
```diff
@@ -23,6 +23,12 @@
 		"fr": "",
 		"ko": ""
 	},
+	"need_to_set_acct_password": {
+		"en": "You need to set your password to finish setting up your account. Please use the forgot password link below.",
+		"es": "",
```
Comment (longrunningprocess) [inline dictionaries/error.definition.json:28]: Just curious if you have plans for translations yet?  I understand you may still be waiting for approval from @fillup about using Google Translate.  In the meantime, I'll send you a link to the spreadsheet we're using to gather input from our local experts.  I'm thinking you could drop yours in there and Phillip can ping those individuals again.

File: features/login.feature [hunk 2]
```diff
@@ -180,3 +180,30 @@ Feature: User login
     When I provide a username of "BOB_ADAMS"
     And I try to log in with an incorrect password enough times to require a captcha
```
Comment (longrunningprocess) [inline features/login.feature:181]: cool framework!

**Review by longrunningprocess — APPROVED** [review]

File: dictionaries/error.definition.json (hunk 1 above)
Comment (forevermatt) [reply dictionaries/error.definition.json:28]: I haven't pinged @fillup about that yet. I can certainly add these lines to the spreadsheet, thanks.

**Comment by forevermatt** [comment]
@fillup I'm going ahead and merging this. If you would, please take a look at that last new entry in the [login.feature](https://github.com/silinternational/simplesamlphp-module-silauth/pull/9/files#diff-d392af37bbf2fc7b8e3bd9e1000109e8) file at some point to make sure what it describes is the way we want that scenario to work.

**Pull request merged**